.github/workflows/build_navitia_packages_for_release_debian_10.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Navitia Packages For Release
+
+on:
+  pull_request:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    container:
+      image: navitia/debian10_dev
```

Comment:
Why not using a matrix build on the previous script ? So we can do `[navitia/debian8_dev, navitia/debian10_dev]`
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix